src/activities/ActivityUsageEntity.js

```diff
@@ -635,9 +635,11 @@ export class ActivityUsageEntity extends Entity {
 			return;
 		}
 
+		const scoreOutOf = this.scoreOutOf() ? this.scoreOutOf().toString() : undefined;
```

Comment:
@wongvincent tagging you in this PR since you are the last to touch this code. Just confirming on your end that this is a non-issue for your team. I don't expect this to cause issues but want you to be aware. When saving a content activity-usage entity there is no related score information and `scoreOutOf()` will return `undefined` causing issues for us.